client/asset/btc/btc.go

```diff
@@ -3719,7 +3719,7 @@ func (btc *intermediaryWallet) FindRedemption(ctx context.Context, coinID, _ dex
 		return nil, nil, fmt.Errorf("queueFindRedemptionRequest error for redemption %s: %w", outPt, err)
 	}
 
-	go btc.tryRedemptionRequests(ctx, nil, []*findRedemptionReq{req})
+	btc.tryRedemptionRequests(ctx, nil, []*findRedemptionReq{req})
```

Comment:
There seems to be no need to run this as a separate go-routine (and dcr wallet doesn't) because we are waiting on the result below anyway. There is a note in `tryRedemptionRequests` func that might be somewhat related:
```
// Do we really want to do this? Mempool could be huge.
searchDur := time.Minute * 5
```

Reply:
moved here: https://github.com/decred/dcrdex/pull/2227/files#r1138098479